customer_outstanding_statement/report/customer_outstanding_statement.py

```diff
@@ -1,6 +1,5 @@
 # -*- coding: utf-8 -*-
-# Copyright 2017 Eficent Business and IT Consulting Services S.L.
```

Comment:
Please, don't delete copyright history if not strictly necessary.

Reply:
remove # -*- coding: utf-8 -*- everywhere